tools/migration-remover/src/template.rs

```diff
@@ -0,0 +1,14 @@
+use axelar_wasm_std::migrate_from_version;
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{DepsMut, Empty, Env, Response};
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+#[migrate_from_version("0.0")]
+pub fn migrate(
+    deps: DepsMut,
+    _env: Env,
+    _msg: Empty,
+) -> Result<Response, axelar_wasm_std::error::ContractError> {
+    Ok(Response::default())
+}
```

Comment:
don't we still want to set the version?

Reply:
its done with the macro `#[migrate_from_version("0.0")]`, the actual version is replaced from contract's cargo.toml